src/bus/webSocketConnection.js

```diff
@@ -107,6 +98,34 @@ class WebSocketConnection {
     this._webSocket.close();
   }
 
+  /**
+   * Handles WebSocket errors.
+   * The `ws` library also closes the socket when an error occurs.
+   * Since the socket connection closes, the jsonRpcId and message handlers are cleared
+   */
+  onError = (error) => {
+    this._messageHandlers = {};
+
+    console.log(error);
+  };
+
+  /**
+   * Handles messages sent from the Message Bus WebSocket connection.
+   */
+  onMessage = (message) => {
```

Comment:
Need to add info on the `message` parameter. And what do you think about making this a "private" method? Users shouldn't really be calling it